src/OpenApi/src/PACKAGE.md

```diff
@@ -0,0 +1,54 @@
+## About
+
+Microsoft.AspNetCore.OpenApi is a NuGet package that provides built-in support for generating OpenAPI documents from minimal or controller-based APIs in ASP.NET.
+
+## Key Features
+
+* Supports generating an OpenAPI document at runtime exposed via an endpoint (`/openapi/{documentName}.json`)
```

Comment:
`documentName` doesn't mean anything to me. Maybe it's a common thing for OpenAPI?

Reply:
The endpoint is parameterized because it is possible for a single app to have multiple OpenAPI documents (for example for different versions of an API or for public/internal versions). I'll update the description a bit to clarify this.